doc/UsersGuide/source/CustomizingTheWorkflow/PhysicsSchemes.rst

```diff
@@ -0,0 +1,134 @@
+.. _PhysicsSchemes:
+
+***********************
+Physics Schemes in HAFS
+***********************
+
+This chapter contains information regarding the various physics schemes in HAFS.
+
```

Comment:
@jdkublnick I think it would make most sense to put the Table comparing HFSA and HFSB physics here in the introduction.